extensions/wikia/WikiaInYourLang/WikiaInYourLang.hooks.php

```diff
@@ -9,12 +9,12 @@
 class WikiaInYourLangHooks {
 
 	/**
-	 * Add JS module to the output
+	 * Add JS assets package to the output
 	 * @param  \OutputPage $out  An output object passed from a hook
 	 * @return bool
 	 */
 	public static function onBeforePageDisplay( \OutputPage $out ) {
-		$out->addModules( 'ext.wikiaInYourLang' );
+		\Wikia::addAssetsToOutput( 'wikia_in_your_lang_js' );
```

Comment:
Is this how we're supposed to be doing this now? I feel like there's 100 ways to do add assets. :grimacing: